public/stylesheets/site/2.0/17-zone-home.css

```diff
@@ -74,12 +74,18 @@ user home, collections and challenges home, tag home, admin comms home
 /*(with filters)*/
 
 /* wrap long text in Firefox (applied to inbox only to avoid weirdness on all forms in IE) */
-#inbox-form {
-  float: left;
+form.inbox {
```

Comment:
Change to the class selector as long as we're adding it, and remove `float: left` because it already has that style from `.dashboard > form`. Bad past!me. No cookie.